### PR TITLE
Change schedule request flow

### PR DESCRIPTION
**Pull request by nuxzero**

### CODE REVIEW

File: src/main/java/co/omise/Example.java [hunk 1]
```diff
@@ -181,7 +186,7 @@ void destroyCustomer() throws IOException, OmiseException, ClientException {
     void listAllDisputes() throws IOException, OmiseException, ClientException {
         Request<ScopedList<Dispute>> request = new Dispute.ListRequestBuilder().build();
         ScopedList<Dispute> disputes = client().sendRequest(request);
-        System.out.printf("no. of disputes: %d", disputes.getTotal());
+        System.out.printf("total no. of disputes: %d", disputes.getTotal());
```
Comment (pitiphong-p) [inline src/main/java/co/omise/Example.java]: Why do we need to print these?

File: src/test/java/co/omise/requests/ScheduleRequestTest.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+        assertEquals("schd_test_57s33hm9fg1pzcqihxs", schedule.getId());
+        assertEquals(1, schedule.getEvery());
+        assertEquals(SchedulePeriod.month, schedule.getPeriod());
+        assertEquals(11, schedule.getNextOccurrenceDates().size());
+    }
+
+    @Test
+    public void testDelete() throws IOException, OmiseException {
+        String scheduleId = "schd_test_57s33hm9fg1pzcqihxs";
```
Comment (Namiii) [inline src/test/java/co/omise/requests/ScheduleRequestTest.java]: I think `scheduleId` can be extracted out of the methods and used in all test and also be used on line 65

**Comment by Namiii** [comment]
@nuxzero I think you need to do a rebase with `v3.0` as the changes to Customer will break your code

File: src/main/java/co/omise/Example.java (hunk 1 above)
Comment (nuxzero) [reply src/main/java/co/omise/Example.java]: Not sure about this @chakrit Do we still need the print out in the example code krub.

File: src/test/java/co/omise/live/LiveScheduleRequestTest.java [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+    private Client client;
+
+    @Before
+    public void setup() throws Exception {
+        client = getLiveClient();
+    }
+
+    @Test
+    @Ignore("only hit the network when we need to.")
+    public void testLiveScheduleListGet() throws IOException, OmiseException {
```
[the remark this answers is not in the document]
Comment (nuxzero) [reply src/test/java/co/omise/live/LiveScheduleRequestTest.java]: OK, I'll add a test retrieve schedule list with the option.

File: src/test/java/co/omise/requests/ScheduleRequestTest.java (hunk 2 above)
Comment (nuxzero) [reply src/test/java/co/omise/requests/ScheduleRequestTest.java]: That right, I'll change

File: src/main/java/co/omise/Example.java (hunk 1 above)
Comment (pitiphong-p) [reply src/main/java/co/omise/Example.java]: Oh I just saw that it's in Example. I think it's ok then.

Sorry for another false alarm

File: src/main/java/co/omise/Example.java [hunk 4]
```diff
@@ -181,7 +186,7 @@ void destroyCustomer() throws IOException, OmiseException, ClientException {
     void listAllDisputes() throws IOException, OmiseException, ClientException {
```
[the remark this answers is not in the document]
Comment (nuxzero) [reply src/main/java/co/omise/Example.java]: Sound good, @Namiii @chakrit What do you about this?

[the remark this answers is not in the document]
Comment (Namiii) [reply src/main/java/co/omise/Example.java]: I don't mind it 👍

[the remark this answers is not in the document]
Comment (chakrit) [reply src/main/java/co/omise/Example.java]: The containing class already defines the context. I think in Java, we should not give in to verbosity so easily 😂

**Comment by nuxzero** [comment]
@pitiphong-p @Namiii Changed follow your request changes.

File: src/main/java/co/omise/Example.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (pitiphong-p) [reply src/main/java/co/omise/Example.java]: Yeah you're right. I'll try to carefully read the filename first next time

**Pull request merged**